### ISSUE TITLE
Branding for CherryPick 

### ISSUE THREAD

**Issue by 4ioskd** [kokonect-link/cherrypick#16]
・Summary
Do you have a plan to write README and create brand assets for CherryPick?

・Tell us what the suggestion is
In general,README is a front page of git repo to introduce your project. But CherryPick's one is based on and just copied partially from misskey-dev/misskey's one.If you are interested in a brand management of CheeryPcik, I recommend writing its original description and creating brand assets such as a logo,colors and so on.

**Comment by noridev** [comment]
There are plans to add new README and logos that characterize CherryPick.
However, there are no concrete plans for what the logo will be as yet.
I will revise the README.md and DIFFERENCE.md as plans are developed.

**Comment by 4ioskd** [comment]
Thank you for your reply quickly!
If you have no concrete plans,it is good for posting a note about asking an idea of the logo or recruiting logo creator.I think.

**Comment by noridev** [comment]
Yes. I think it's okay to solicit ideas. For now, I'll try it on my own, and if it doesn't work, I'll try :)
